tests/test_contingency.py

```diff
@@ -0,0 +1,131 @@
+import xarray as xr
+import pytest
+import scores
+
+from tests.assertions import assert_dataarray_equal
```

Comment:
Use xr.testing, so that we can eventually remove this from the codebase

Reply:
Done, thanks for your other MR.